bitnami/elasticsearch/templates/data-statefulset.yaml

```
@@ -71,7 +71,7 @@ spec:
             - /bin/bash
             - -ec
             - |
-              sysctl -w vm.max_map_count=262144 && sysctl -w fs.file-max=65536
+              sysctl -w vm.max_map_count={{ .Values.sysctlImage.vmMaxMapCount }} && sysctl -w fs.file-max={{ .Values.sysctlImage.fsFileMax }}
```

Would it make sense to force integer conversion?
```suggestion
              sysctl -w vm.max_map_count={{ int .Values.sysctlImage.vmMaxMapCount }} && sysctl -w fs.file-max={{ int .Values.sysctlImage.fsFileMax }}
```